### PR TITLE
Some minor additional tests using a new CLI test structure.

### PR DESCRIPTION
**Pull request by Gonthim**
Further refactoring to get all of the CLI tests using the structure to follow.

### CODE REVIEW

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessBuilder.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package org.jboss.as.test.integration.management.cli;
```
Comment (jamezp) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessBuilder.java]: Copyright needed

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessWrapper.java [hunk 2]
```diff
@@ -0,0 +1,363 @@
+package org.jboss.as.test.integration.management.cli;
```
Comment (jamezp) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessWrapper.java]: Same here, copyright needed.

File: testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/CtrlCTestCase.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package org.jboss.as.test.integration.management.cli;
```
Comment (jamezp) [inline testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/CtrlCTestCase.java]: Copyright

**Comment by jamezp** [comment]
Adding a fixme label as the test failures seem related to the changes.

**Comment by Gonthim** [comment]
The CI build is using the wrong commit. Updating the branch seems to have confused it. Closing and re-pulling.

**Pull request closed without merging**